pkg/confluence/api.go

```diff
@@ -514,17 +523,6 @@ func (api *API) UpdatePage(page *PageInfo, newContent string, minorEdit bool, ve
 		}
 	}
 
-	labels := []map[string]interface{}{}
-	for _, label := range newLabels {
-		if label != "" {
-			item := map[string]interface{}{
-				"prexix": "global",
```

Comment:
looks like there was a typo here anyway. 

Is the change backwards compatible with this one in mind?